app/screens/post_options/post_options.js

```diff
@@ -386,18 +386,16 @@ export default class PostOptions extends PureComponent {
     render() {
         const {deviceHeight} = this.props;
         const options = this.getPostOptions();
-        const initialPosition = (options.length + 1) * OPTION_HEIGHT;
         const marginFromTop = deviceHeight - BOTTOM_MARGIN - ((options.length + 1) * OPTION_HEIGHT);
 
         return (
-            <View style={style.flex}>
+            <View style={style.container}>
                 <SlideUpPanel
                     allowStayMiddle={false}
-                    alwaysCaptureContainerMove={true}
                     ref={this.refSlideUpPanel}
                     marginFromTop={marginFromTop}
                     onRequestClose={this.close}
-                    initialPosition={initialPosition}
+                    initialPosition={325}
```

Comment:
Should this change based on the number of items or on screen size? With the number of items we have in the list, will it ever leave an empty space at the bottom of the list?

Reply:
the spec defined this to be an initial of 325 points